tests/phpunit/CRM/MembershipExtras/Hook/ValidateForm/MembershipTypeTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * Class CRM_MembershipExtras_Hook_ValidateForm_MembershipTypeTest
+ *
+ * @group headless
+ */
+class CRM_MembershipExtras_Hook_ValidateForm_MembershipTypeTest extends BaseHeadlessTest {
+
+  /**
+   * Tests validate fixed period with invalid duration.
```

Comment:
I don't see the purpose of these docblocks, they don't add any value and merely repeat the test function name, I would get rid of em.

Reply:
@omarabuhussein  It makes function easier to read.

Reply:
I don't think it does